.rubocop.yml

```
@@ -0,0 +1,195 @@
+# use the shared Yast defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
```

For master we should prefer the newer Rubocop (0.71.0), with the defaults stored in the [rubocop-0.71.0_yast_style.yml](https://github.com/yast/yast-devtools/blob/master/ytools/y2tool/rubocop-0.71.0_yast_style.yml) file.

```suggestion
  /usr/share/YaST2/data/devtools/data/rubocop-0.71.0_yast_style.yml
```